uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/app-dashboard.module.js

```diff
@@ -26,8 +26,9 @@ define([
               })
               .state('tasks', {
                 url: '/tasks',
-                controller: 'TaskListCtrl',
+                controller: 'TaskListController',
                 templateUrl: config.path.TPL + 'dashboard/tasks.html?v=' + (new Date().getTime()),
+                controllerAs: 'List',
```

Comment:
I don't really think we use capital letters in controllerAs, this becomes a variable in templates and capitalised variabled usually mean classes.

Reply:
Changed to small caps.